src/theme/NotFound.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Translate, { translate } from '@docusaurus/Translate';
+import { PageMetadata } from '@docusaurus/theme-common';
+import Layout from '@theme/Layout';
+import clsx from 'clsx';
+
+import { HeroPattern } from '@features/LandingPage/HeroHeader/HeroPattern';
+import styles from './NotFound.module.css';
+
+export default function NotFound() {
+  return (
+    <>
+      <PageMetadata
+        title={translate({
+          id: 'theme.NotFound.title',
+          message: 'Page Not Found',
+        })}
+      />
+      <Layout>
+        <main className="container margin-vert--xl">
+          <div className="row">
+            <div className={clsx('col col--8 ' + styles.centerColumn)}>
+              <h1 className="hero__title">
+                <Translate id="theme.NotFound.title" description="The title of the 404 page">
+                  404: Page Not Found
+                </Translate>
+              </h1>
+              <p>
+                If you navigated here from a broken link on the OpenFGA website, please{' '}
+                <a href="https://github.com/openfga/openfga.dev/issues" target="_blank" rel="noreferrer">
```

Comment:
Instead of using a directly, should we use Link component instead?

Reply:
+1 It's an external link so it doesn't benefit a ton from using the `<Link />` but this would add consistency in the code.

Good ol' swizzles..

Reply:
The good thing is NotFound is listed as safe to swizzle, so we don't need to worry about it when upgrading Docusaurus version.

Reply:
Fixed,thanks!